R/produce_slf_ch_tests.R

```diff
@@ -0,0 +1,29 @@
+#' Care Home All Episodes Tests
+#'
+#' @description Produce the test for the Care Home
+#'
+#' @param data new or old data for testing summary flags
+#' (data is from [get_source_extract_path(year, "CH")])
+#'
+#' @return a dataframe with a count of each flag.
+#'
+#' @export
+#' @family social care test functions
+produce_slf_ch_tests <- function(data) {
```

Comment:
```suggestion
produce_slf_ch_year_specific_tests <- function(data) {
```
Rename to something like this if this is the year specific care home tests?